src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -244,52 +244,50 @@ describe('Language selection', () => {
 
   test('translatable fields appear in selected languages', async () => {
     const { user } = renderComponent();
+    await actWait();
     const languageSelector = await screen.findByTestId(
       formLanguageSelectorTestId
     );
-
     // Select Swedish (with Finnish that is already selected)
     const sv = await within(languageSelector).findByRole('checkbox', {
       name: /ruotsi/i,
     });
     await user.click(sv);
+    await actWait();
     await waitFor(() => {
       expect(sv).toBeChecked();
     });
 
     for (const labelText of translatableFieldLabels) {
-      await waitFor(
-        () => {
-          expect(
-            screen.getByLabelText(
-              new RegExp(labelText.source + / \(FI\)/.source)
-            )
-          ).toBeInTheDocument();
-          expect(
-            screen.getByLabelText(
-              new RegExp(labelText.source + / \(SV\)/.source)
-            )
-          ).toBeInTheDocument();
-        },
-        { timeout: 5_000 }
-      );
+      await waitFor(() => {
+        expect(
+          screen.getByLabelText(new RegExp(labelText.source + / \(FI\)/.source))
+        ).toBeInTheDocument();
+      });
+      await waitFor(() => {
+        expect(
+          screen.getByLabelText(new RegExp(labelText.source + / \(SV\)/.source))
+        ).toBeInTheDocument();
+      });
     }
 
     // Unselect Finnish
     const fi = await within(languageSelector).findByRole('checkbox', {
       name: /suomi/i,
     });
     await user.click(fi);
+    await actWait();
```

Comment:
actWait should not be needed. How about moving it till the end of the test if it is here for the act-warnings?
https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning

Reply:
Removed most uses of `await actWait();` in 5fe6477e98db1ed3a2d774936736a99dc66aa63f